tests/func/test_remove.py

```diff
@@ -69,3 +69,35 @@ def test_cmd_remove(tmp_dir, dvc):
     assert main(["remove", stage.addressing, "--outs"]) == 0
     assert not (tmp_dir / stage.relpath).exists()
     assert not (tmp_dir / "foo").exists()
+
+
+def test_cmd_remove_gitignore_single_stage(tmp_dir, scm, dvc, run_copy):
+    stage = dvc.run(
+        name="my", cmd='echo "hello" > out', deps=[], outs=["out"],
+    )
+
+    assert (tmp_dir / ".gitignore").exists(), ".gitignore does not exist"
+
+    assert main(["remove", stage.addressing]) == 0
+    assert not (tmp_dir / stage.relpath).exists(), "Pipeline file does exists"
```

Comment:
It is pretty self explanatory, we tend to not use assert messages instead trying to make the code self-explanatory, so for consistency let's just:
```suggestion
    assert not (tmp_dir / stage.relpath).exists()
```